ci-operator/step-registry/baremetalds/e2e/ovn/dualstack/baremetalds-e2e-ovn-dualstack-workflow.yaml

```diff
@@ -0,0 +1,20 @@
+workflow:
+  as: baremetalds-e2e-ovn-dualstack
+  steps:
+    env:
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v4v6
+        NETWORK_TYPE=OVNKubernetes 
+    pre:
+      - ref: baremetalds-packet-setup
+      - ref: ipi-install-rbac
+      - ref: ovn-conf-ipsec-manifest
```

Comment:
What's the need of this? All the current `e2e-metal-ipi-ovn-dualstack` tests defined across the various repos are simpler, ie https://github.com/openshift/release/blob/d8778edfa4d02168d6317c9cdc6dd95dcfe37891/ci-operator/config/openshift/baremetal-operator/openshift-baremetal-operator-master.yaml#L89-L96

Reply:
+1. That makes sense.